.changeset/lemon-pens-move.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit/localize-tools': patch
+---
+
+Updated xmldom dependency. Minor change to XML attribute formatting can be expected.
```

Comment:
Sorry I haven't tested this, but do you happen to know if older versions of `.xlf` files would be read correctly? It seems now `>` is also being escaped to `&gt;` so wondering if this might be a breaking change for users. Since it's only for the text nodes so I imagine it probably shouldn't be breaking.

Reply:
Yeah, both are valid XML serializations (only the < technically needs to be escaped). I  manually tested that the older serialization is parsed in the same way, and there should already be some coverage in our integration tests also, because I only updated the goldens, not the inputs (which includes existing xlf files).